src/pymor/parameters/base.py

```diff
@@ -572,6 +572,55 @@ def sample_randomly(self, count=None):
         else:
             return [get_param() for _ in range(count)]
 
+    def sample_logarithmic_uniformly(self, counts, base):
+        """Logarithmically uniform sample |parameter values| from the space.
+
+        Parameters
+        ----------
+        counts
+            Number of samples to take per parameter and component
+            of the parameter. Either a dict of counts per |Parameter|
+            or a single count that is taken for each parameter in |Parameters|.
+
+        Returns
+        -------
+        List of |parameter value| dicts.
+        """
+        if not isinstance(counts, dict):
+            counts = {k: counts for k in self.parameters}
+
+        logspaces = tuple(np.logspace(np.log10(self.ranges[k][0]) / np.log10(base),
+                                      np.log10(self.ranges[k][1]) / np.log10(base),
+                                      num=counts[k], base=base)
```

Comment:
I feel that this operation is independent of `base`, and so it should be possible to remove `base` and use `np.geomspace` instead of `np.logspace`.

Something similar might be possible in `sample_logarithmic_randomly`.

Reply:
Hii @sdrave, I have removed base and applied np.geomspace per your suggestions. I hope I've managed it this time.